code/modules/power/floodlight.dm

```diff
@@ -147,8 +147,26 @@
 
 /obj/machinery/power/floodlight/Initialize(mapload)
 	. = ..()
+	RegisterSignal(src, COMSIG_OBJ_PAINTED, TYPE_PROC_REF(/obj/machinery/power/floodlight/, on_color_change))  //update light color when color changes
```

Comment:
```suggestion
	RegisterSignal(src, COMSIG_OBJ_PAINTED, TYPE_PROC_REF(/obj/machinery/power/floodlight, on_color_change))  //update light color when color changes
```

otherwise spooky stuff could happen in 515... maybe not but let's just be safe